Octokit/Models/Response/StringEnum.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using Octokit.Internal;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    [SuppressMessage("Microsoft.Naming", "CA1711:IdentifiersShouldNotHaveIncorrectSuffix")]
+    public struct StringEnum<TEnum> : IEquatable<StringEnum<TEnum>>
+        where TEnum : struct
+    {
+        private readonly string _value;
+
+        private TEnum? _parsedValue;
+
+        public StringEnum(TEnum parsedValue)
+        {
+            if (!Enum.IsDefined(typeof(TEnum), parsedValue))
+            {
+                throw GetArgumentException(parsedValue.ToString());
+            }
+
+            // Use the SimpleJsonSerializer to serialize the TEnum into the correct string according to the GitHub Api strategy
+            _value = new SimpleJsonSerializer().SerializeEnum(parsedValue as Enum);
```

Comment:
I was trying to avoid changing too much about the current deserializer code.  Both the underlying `GitHubSerializationStrategy` (which implements an interface) and the actual `SimpleJsonSerializer` both use instance methods rather than statics.